### ISSUE TITLE
CSRF utility breaks on _multipart uploads

### ISSUE THREAD

**Issue by mikericher** [sergiodxa/remix-utils#46]
I had a form with the <AuthenticitiyTokenInput /> and an <input type="file" name="picture" />

in the action if the "await verifyAuthenticitiyToken(request, session)" is before the form the unstable_parseMultipartFormData function is unable to process the supplied form with:

Tried to parse multipart file upload for field "picture" but no uploadHandler was provided. Read more here: https://remix.run/api/remix#parseMultipartFormData-node

if I do the verify after the multipart form, the verify dies..

I assume the utility is just not multipart aware?  or I could be going down another rabbit hole...

**Comment by mikericher** [comment]
oops.. markdown:

I had a form with the AuthenticityToken and an input field with type="file" name="picture"

**Comment by mikericher** [comment]
In order for me to get this to work, I had to remove the request.clone().formData from the csrf utilities, and instead of handing the 'request' to the verify function, I had to pass the formData.

Apparently something to do with clone() breaking large file multipart requests.

with the remix-utils version, if you verify before multipart.. it breaks multipart... and if you verify after multipart.. verify breaks with a timeout / unprocessableEntity entity.

I have not quite figured out how to break early on a file upload if it's a failed 'csrf.  Only thing I can think of here, is to have the "input" be at the top of the form, and in the upload handler verify and throw, before processing the file.
